spec/dummy/db/schema.rb

```diff
@@ -112,7 +112,7 @@
     t.text     "customer_address"
     t.text     "business_address"
     t.integer  "setup_fee"
-    t.decimal  "tax_percent"
+    t.integer  "tax_percent"
```

Comment:
Could you revert the changes to `db/schema.rb` please? This accidentally changes `tax_percent` back to an integer.